File: EASY/TWO SUM/twoSum.java
class twoSum{
    public int[] twosum(int[] nums, int target){
        for(int i=0; i<nums.length; i++){
            for(int j=i+1; j<nums.length; j++){
                if(nums[i]+nums[j]==target){
                    return new int[]{i,j};
                }
            }
        }
        return null;
    }
    public static void main(String[] args){
        twoSum ob = new twoSum();
        int[] nums = {2,7,11,15};
        int target = 9;
        int[] result = ob.twosum(nums, target);
        for(int i=0; i<result.length; i++){
            System.out.print(result[i] + ", ");
        }
    }
}

File: EASY/TWO SUM/twosum.c
#include <stdio.h>
#include <stdlib.h>
int* twosum(int* nums, int numsSize, int target, int* returnSize){
    int* result = (int*) malloc(2*sizeof(int));
    *returnSize = 2;

    int i, j;
    for(i=0; i<numsSize; i++){
        for(j=i+1; j<numsSize; j++){
            if(nums[i] + nums[j] == target){
                result[0] = i;
                result[1] = j;
                return result;
            }
        }
    }
    return NULL;
}
void main(){
    int nums[] = {2,7,11,15};
    int length = sizeof(nums)/sizeof(nums[0]);
    int target = 9;
    int resultSize;
    int* result = twosum(nums, length, target, &resultSize);
    printf("[%d,%d]", result[0], result[1]);
}